pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -528,6 +550,40 @@ func (a FeatureGateAnonymizer) Marshal(_ context.Context) ([]byte, error) {
 	return runtime.Encode(serializer, a.FeatureGate)
 }
 
+// IngressAnonymizer implements serialization with marshalling
+type ImageRegistryAnonymizer struct {
+	*registryv1.Config
+}
+
+// Marshal implements serialization of Ingres.Spec.Domain with anonymization
+func (a ImageRegistryAnonymizer) Marshal(_ context.Context) ([]byte, error) {
+	a.Spec.HTTPSecret = anonymizeString(a.Spec.HTTPSecret)
```

Comment:
What if we would just remove all Storage ? Can here be any other types, like pvc ?
And what about HttpClaim, is it something to remove as well ?

Reply:
If we remove Storage, we won't be able to know if someone uses a storage parameter.